src/ui/public/styles/ui_select.less

```diff
@@ -25,5 +25,5 @@
 }
 
 .btn-default .ui-select-placeholder {
-  color: tint(@globalColorBlue, 70%);
+  color: @input-color-placeholder;
```

Comment:
Looks like it was a very light blue. Probably because if it's sitting on a button (which itself is very blue) it contrasted better than a dark gray that is normally on white.